pkg/controllers/provisioning/v1alpha1/vars.go

```diff
@@ -0,0 +1,35 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"time"
+)
+
+const (
+	TimeFormat = time.RFC3339
```

Comment:
Package scope variables should be used judiciously. TimeFormat and TTL seconds should probably be file scoped to where you're using them.

Reply:
I was thinking that the time format may be used by other packages in the future. I can scope it more tightly though.